coder-sdk/env.go

```diff
@@ -94,6 +95,24 @@ func (c Client) CreateEnvironment(ctx context.Context, req CreateEnvironmentRequ
 	return &env, nil
 }
 
+// Template is used to configure a new environment from a repo.
```

Comment:
This will show up in the `coder-sdk` godoc, so maybe we should include an `alpha` note in the doc comment.